src/frontend/js/modules/authentication.js

```diff
@@ -8,13 +8,16 @@ angular.module('esn.authentication', ['esn.http', 'ui.router'])
       search: $location.search()
     };
 
-    if (currentLocation.path !== authCallbackPath) {
+    if (currentLocation.path !== authCallbackPath && currentLocation.path !== authCallbackPath + '/') {
```

Comment:
```suggestion
    if (currentLocation.path !== authCallbackPath && currentLocation.path !== `${authCallbackPath}/`) {
```

Reply:
I also replaced it on other line I used it.